### PR TITLE
Issue 151: Create range slider element

### PR DESCRIPTION
**Pull request by Torboto**
https://github.com/jegelstaff/formulize/issues/151

### CODE REVIEW

**Comment by Torboto** [comment]
@eikonos This is ready to be reviewed.

File: mainfile.php [hunk 1]
```diff
@@ -4,14 +4,152 @@
 *
 * Be careful if you are changing data's in this file.
 *
+* @copyright	http://www.xoops.org/ The XOOPS Project
```
Comment (eikonos) [inline mainfile.php]: Changes to the mainfile should be reverted...

File: modules/formulize/class/newsliderElement.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        $update_script .= "document.getElementById('rangeValue').value=val;}";
+        $update_script .= "</script>";
+
+        if($isDisabled) {
+            $renderedValue = $form_slider_value->render();
+            $form_ele = new XoopsFormLabel($caption, "$renderedValue");
+        } else {
+            $renderedSlider = $form_slider->render();
+            $renderedValue = $form_slider_value->render();
+            $form_ele = new XoopsFormLabel($caption, "<nobr>$renderedSlider $renderedValue</nobr>.$update_script");
```
Comment (eikonos) [inline modules/formulize/class/newsliderElement.php]: Can you change ".$update_script" to " $updatescript" because the . shows up on the page?

**Comment by Torboto** [comment]
@eikonos Fixed slider not loading the DB value when editing an entry. Should be good to go!

**Pull request merged**